### PR TITLE
MAISTRA-566 use multitenant=true as default

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,32 @@ bash <(curl -L https://git.io/getLatestKialiOperator) --uninstall-mode true --op
...
 For more details on uninstalling the Kiali operator, see the [Kiali documentaton](https://www.kiali.io/documentation/getting-started/#_uninstall_kiali_operator_and_kiali).
 
+## Multitenancy
+
+The operator installs a control plane configured for multitenancy.  This installation reduces the scope of the control plane
+to only those projects/namespaces listed in a `ServiceMeshMemberRoll`.  After installing the control plane, create/update
+a ServiceMeshMemberRoll resource with the project/namespaces you wich to be part of the mesh.  The name of the
+ServiceMeshMemberRoll resource must be named `default`.  The operator will configure the control plane to watch/manage pods
+in those projects/namespaces and will configure the project/namespaces to work with the control plane.
+
```
Comment (jwendell) [inline README.md:114]: Does the order of creation matters? What happens if the user first creates the namespace and deploy their services and only add that namespace to a `smmr` later? Worth to document/highlight this case?

**Review by jwendell — APPROVED** [review]
LGTM, just left a comment in README

File: README.md (hunk 1 above)
Comment (knrc) [reply README.md:114]: Yes, if the application is deployed before the namespace is set up then it will not be configured for the service mesh.

Comment (knrc) [reply README.md:114]: There's a typo above, wich should be wish?

File: README.md [hunk 2]
```diff
@@ -103,6 +103,32 @@ bash <(curl -L https://git.io/getLatestKialiOperator) --uninstall-mode true --op
 
 For more details on uninstalling the Kiali operator, see the [Kiali documentaton](https://www.kiali.io/documentation/getting-started/#_uninstall_kiali_operator_and_kiali).
 
+## Multitenancy
+
+The operator installs a control plane configured for multitenancy.  This installation reduces the scope of the control plane
+to only those projects/namespaces listed in a `ServiceMeshMemberRoll`.  After installing the control plane, create/update
+a ServiceMeshMemberRoll resource with the project/namespaces you wich to be part of the mesh.  The name of the
```
Comment (brian-avery) [inline README.md]: you wish

File: deploy/examples/maistra_v1_servicemeshcontrolplane_cr_minimal_cni.yaml [hunk 3]
```diff
@@ -80,3 +78,22 @@ spec:
     # to disable tracing (i.e. jaeger)
     tracing:
       enabled: false
+            jaeger:
+        # simple, all-in-one strategy
```
Comment (brian-avery) [inline deploy/examples/maistra_v1_servicemeshcontrolplane_cr_minimal_cni.yaml:82]: is the spacing off?

**Review by brian-avery — APPROVED** [review]

File: deploy/examples/maistra_v1_servicemeshcontrolplane_cr_minimal_cni.yaml (hunk 3 above)
Comment (rcernich) [reply deploy/examples/maistra_v1_servicemeshcontrolplane_cr_minimal_cni.yaml:82]: yep

File: README.md (hunk 2 above)
Comment (rcernich) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (rcernich) [reply README.md:114]: added a note about auto-injection

**Comment by rcernich** [comment]
Okay, I've addressed these comments.

File: README.md (hunk 1 above)
Comment (jwendell) [reply README.md:114]: So, if the user messes with the order, what should he do? Will killing the pods force the injection to happen? Must he delete the whole namespace and recreate everything again?

Comment (rcernich) [reply README.md:114]: yes, injection occurs on pod creation.  this should be covered by istio docs.

**Comment by rcernich** [comment]
Added commit for MAISTRA-580, which fixes networking issues with galley, sidecar-injector, and ingressgateway.

**Comment by rcernich** [comment]
removed commit that addresses MAISTRA-580

**Review by knrc — APPROVED** [review]

**Comment by knrc** [comment]
Holding for now

**Pull request merged**